prime-router/build.gradle.kts

```diff
@@ -673,9 +673,9 @@ dependencies {
         exclude(group = "com.azure", module = "azure-core")
         exclude(group = "com.azure", module = "azure-core-http-netty")
     }
-    implementation("org.apache.logging.log4j:log4j-api:[2.17.1,)")
-    implementation("org.apache.logging.log4j:log4j-core:[2.17.1,)")
-    implementation("org.apache.logging.log4j:log4j-slf4j-impl:[2.17.1,)")
+    implementation("org.apache.logging.log4j:log4j-api:2.17.1")
+    implementation("org.apache.logging.log4j:log4j-core:2.17.1")
+    implementation("org.apache.logging.log4j:log4j-slf4j-impl:2.17.1")
```

Comment:
@jorg3lopez did you update these references yourself?

Reply:
I'm keeping it as it is in master. I'll make the changes soon. Don't want to include extra things on the PR.